### DISCUSSION TITLE
PrimeFaces 13 compatible templates

### DISCUSSION THREAD

**Discussion by jepsar** [primefaces/community#50] [PrimeFaces Templates] [answered]
When will the Premium (!) templates be updated to PrimeFaces 13 (releases already two months ago)? Over a month ago, on the old forum, I was told support would be added in a few weeks. There have been numerous questions about this on Discord as well.

**Comment by atkntepe** [comment]
Hi, for Primefaces templates, our first plan is to update them to PF13. We had some changes in the PF team that caused some delays. We will be updating them as soon as possible. Thank you for your support 🚀

**Reply by jepsar** [reply]
Three weeks have passed. What do you guys consider as soon as possible? Do you have a date for us?

**Reply by jepsar** [reply]
@atakantepe any news here? This is taking way too long for a Premium product.

**Reply by jepsar** [reply]
@cagataycivici @mertsincan @atakantepe Hi! When can we expect PrimeFaces 13 compatible templates? PrimeFaces 13 was released three months ago.

**Reply by jepsar** [reply]
@cagataycivici @mertsincan @atakantepe Are you alive? This is no way to handle support on premium products.

**Comment by onursenture** [comment, marked as answer]
![primefaces-13-premium-o](https://github.com/primefaces/community/assets/244957/e47a4ca7-a832-47da-bbc9-133bb160cf82)

Updates have been rolled out for our premium PrimeFaces templates to align with PrimeFaces 13! Acquire the most recent versions from PrimeStore at https://primefaces.org/store/

**Reply by jepsar** [reply]
Thank you so much! @mertsincan
